include/config.h

```diff
@@ -117,7 +117,10 @@
 //#define CUSTOM_DEBUG
 // Include Puppyprint, a display library for text and large images. Also includes a custom, enhanced performance profiler.
 //#define PUPPYPRINT
-//#define PUPPYPRINT_DEBUG 0
+#define PUPPYPRINT_DEBUG 0
```

Comment:
should this be off by default?

Reply:
Yeah that's the correct setting. It being commented out was a mistake.